core/test.go

```diff
@@ -45,3 +48,19 @@ func (i *TestEngine) FlagSet() *pflag.FlagSet {
 func (i *TestEngine) Name() string {
 	return "test"
 }
+
+// Problem is a helper struct to Unmarshal problem.Problem
+type Problem struct {
+	Title  string `json:"title"`
+	Status int    `json:"status"`
+	Detail string `json:"detail"`
+}
+
+// ErrorToProblem returns a Problem generated from a problem.Problem
+// problem.Problem doesn't expose its fields
+func ErrorToProblem(err error) Problem {
```

Comment:
I don't like these poluting the global namespace. The `problem` can perhaps be made private?
Or a few helper methods:
* `TestAssertProblemErrStatus(err error, status int) bool`
* `TestAssertProblemErrTitle(err error, title string) bool`
* `TestAssertProblemErrMessage(err error, message string) bool`

Reply:
These have been moved to `nuts-node/test`. Since the conversion from from `error` to `problem.Probelm` now takes place within the test method, I've added a check to make sure the error actually is a problem before verifying the field's specified value.